build/pxc-entrypoint.sh

```diff
@@ -385,7 +385,10 @@ if [ -z "$CLUSTER_JOIN" ] && [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			GRANT ALL ON *.* TO 'operator'@'${MYSQL_ROOT_HOST}' WITH GRANT OPTION ;
 
 			CREATE USER 'xtrabackup'@'%' IDENTIFIED BY '$(escape_special "${XTRABACKUP_PASSWORD}")';
-			GRANT ALL ON *.* TO 'xtrabackup'@'%';
+			GRANT BACKUP_ADMIN, RESOURCE_GROUP_USER, SERVICE_CONNECTION_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'xtrabackup'@'%';
```

Comment:
@inelpandzic @egegunes  I am thinking about 'localhost'  as a host_name for this user, but in this case, the main problem is PITR. We use it in https://github.com/percona/percona-xtradb-cluster-operator/blob/main/cmd/pitr/recoverer/recoverer.go#L42-L43, and it is run as a separate pod. We need to leave "%" or use a separate user for PITR.